action.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: 'Suggest Changes'
+description: 'Create review suggestions from working directory changes'
+inputs:
+  token:
+    description: Access token to make authenticated API calls
+    default: ${{ github.token }}
+  comment:
+    description: The pull request review comment
```

Comment:
should this have a default?

Reply:
I didn't give it a default since it's an optional input. If no comment is provided, none will be left with the review. Do you think it would be better to have a default?

Reply:
ah no that's okay, I thought a comment would be required